recipe/bld.bat

```diff
@@ -13,6 +13,7 @@ cmake -G "NMake Makefiles" ^
       -D TIFF_NAMES=libtiff_i ^
       -D PROJ4_NAMES=proj_i ^
       -D JPEG_NAMES=libjpeg ^
+	  -D GEOTIFF_CSV_DATA_DIR=%LIBRARY_PREFIX%\Library\share\epsg_csv
```

Comment:
Should be `%LIBRARY_PREFIX%\share\epsg_csv` since `LIBRARY_PREFIX` already has the `Library` bit

Reply:
And finish line with `^`